R/autoxgboost.R

```diff
@@ -62,7 +62,9 @@ autoxgboost = function(task, measure = NULL, control = NULL, par.set = NULL, max
   assertNumeric(initial.subsample.range, lower = 0, upper = 1, len = 2, null.ok = TRUE)
   if (!is.null(initial.subsample.range) & (initial.subsample.range[2] <= initial.subsample.range[1]))
     stop("Upper initial subsample range musst be greater (>) than lower ranger")
-  assertChoice(factor.encoder, c("impact", "dummy"))
+  if (is.infinite(impact.encoding.boundary) == TRUE)
+    impact.encoding.boundary = .Machine$integer.max
```

Comment:
what should this line do?

Reply:
Since ````Inf````  is not an integer, i needed a workaround to get a number which cannot be lower than the largest number of factor levels, so I took the largest integer the user's machine has to offer. Another possibility would be to take the maximal number of factor levels of the underlying data + 1, or set up the boundary as a numeric value, which is technically no problem, but makes no sense from the point of view of content. What's the best choice here?

Reply:
drop this:  == TRUE